.github/workflows/nestbuildmatrix.yml

```diff
@@ -102,16 +102,40 @@ jobs:
       - name: "Install dependencies"
         run: |
           sudo apt install python3-docutils
-          wget --progress=dot:mega "https://github.com/errata-ai/vale/releases/download/v2.15.2/vale_2.15.2_Linux_64-bit.tar.gz"
-          echo "7528175c995818bcb88b07574dd2e17c1aad13f5676880f43927bb8f673095aa  vale_2.15.2_Linux_64-bit.tar.gz" | sha256sum -c
-          tar -xzf vale_2.15.2_Linux_64-bit.tar.gz
+          wget --progress=dot:mega 'https://github.com/errata-ai/vale/releases/download/v2.15.2/vale_2.15.2_Linux_64-bit.tar.gz'
+          echo '7528175c995818bcb88b07574dd2e17c1aad13f5676880f43927bb8f673095aa  vale_2.15.2_Linux_64-bit.tar.gz' | sha256sum -c
+          tar -xzf 'vale_2.15.2_Linux_64-bit.tar.gz'
 
       - name: "Run vale..."
         run: |
           echo "Starting vale..."
           ./vale -v
           ./vale doc
 
+  lychee:
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: "Checkout repository content"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: "Install dependencies"
+        run: |
+          #sudo apt install python3-docutils
+          wget --progress=dot:mega 'https://github.com/lycheeverse/lychee/releases/download/v0.8.2/lychee-v0.8.2-x86_64-unknown-linux-gnu.tar.gz'
+          echo '12c27c9b6d551aea9178080d27de75ed26125b130722153399259432f723e606  lychee-v0.8.2-x86_64-unknown-linux-gnu.tar.gz' | sha256sum -c
+          tar -xzf 'lychee-v0.8.2-x86_64-unknown-linux-gnu.tar.gz'
```

Comment:
Any particular reason for the old version, and using a tar.gz? Compare the one-liner ``uses: lycheeverse/lychee-action@v1.2.0``

Reply:
Tried the lychee action, took much more time since the docker container is rebuilt on every invocation. The TarGz is much cheaper and trivial to set up.

Reply:
Latest Lychee version is [v0.10.3](https://github.com/lycheeverse/lychee/releases/tag/v0.10.3), but I think this can be updated later.